### PR TITLE
Update Developer Guide

### PR DESCRIPTION
**Pull request by WilliamRyank**

### CODE REVIEW

File: docs/DeveloperGuide.adoc [hunk 1]
```diff
@@ -12,9 +12,9 @@ ifdef::env-github[]
 :note-caption: :information_source:
 :warning-caption: :warning:
 endif::[]
-:repoURL: https://github.com/se-edu/addressbook-level3/tree/master
+:repoURL: https://github.com/AY1920S1-CS2103T-T13-1/main
 
-By: `Team SE-EDU`      Since: `Jun 2016`      Licence: `MIT`
+By: `CS2013T-T13-1`      Since: `Sep 2019`      Licence: `MIT`
```
Comment (Yen-Peng) [inline docs/DeveloperGuide.adoc]: Amend the team name. Its CS2103T-T13-1.

File: docs/DeveloperGuide.adoc [hunk 2]
```diff
@@ -279,13 +279,13 @@ Refer to the guide <<DevOps#, here>>.
 
 *Target user profile*:
 
-* has a need to manage a significant number of contacts
+* hospital administrative staff who needs to find matches between patients and organ donors
```
Comment (TheRealRavager) [inline docs/DeveloperGuide.adoc]: `needs` change to `need`

File: docs/DeveloperGuide.adoc [hunk 3]
```diff
@@ -279,13 +279,13 @@ Refer to the guide <<DevOps#, here>>.
...
 * prefer desktop apps over other types
 * can type fast
 * prefers typing over mouse input
 * is reasonably comfortable using CLI apps
 
-*Value proposition*: manage contacts faster than a typical mouse/GUI driven app
+*Value proposition*: can find a match between a patient and potential organ donors faster than a typical mouse/GUI driven app
```
Comment (TheRealRavager) [inline docs/DeveloperGuide.adoc]: `can find a match between ...` change to `find matches between ...`

File: docs/DeveloperGuide.adoc [hunk 4]
```diff
@@ -297,33 +297,47 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
...
 |`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
 
-|`* * *` |user |add a new person |
+|`* * *` |user |add a new patient/donor/doctor |
 
-|`* * *` |user |delete a person |remove entries that I no longer need
+|`* * *` |user |delete a patient/donor/doctor |remove entries that I no longer need
 
-|`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
+|`* * *` |user |find a patient/donor/doctor by certain keywords |locate details of persons without having to go through the entire list
```
Comment (TheRealRavager) [inline docs/DeveloperGuide.adoc]: `... doctor by certain keywords ...` change to `... doctor using certain keywords ... `

File: docs/DeveloperGuide.adoc [hunk 5]
```diff
@@ -334,13 +348,31 @@ Use case ends.
 +
 Use case ends.
 
-* 3a. The given index is invalid.
+* 3a. The given Nric is invalid.
```
Comment (TheRealRavager) [inline docs/DeveloperGuide.adoc]: NRIC all caps I think? Cause its an acronym

**Pull request merged**